keps/sig-node/1287-in-place-update-pod-resources/README.md

```diff
@@ -811,7 +824,7 @@ be a race condition where the Kubelet may or may not accept the first resize, de
 it admits the first change before seeing the second. This race condition is accepted as working as
 intended.
 
-The atomic resize requirement should be reevaluated prior to GA, and in the context of pod-level resources.
+The atomic resize requirement should be reevaluated in the context of pod-level resources.
```

Comment:
since this isn't a requirement here, maybe just say `may be reevaluated` ? like I don't think that would block pod level resources either, so probably we do it as a separate kep from either this or that